.github/workflows/ci.yaml

```diff
@@ -37,7 +43,186 @@ jobs:
         with:
           toolchain: stable
           override: true
-      - uses: actions-rs/cargo@v1
+      - name: cargo fetch
+        uses: actions-rs/cargo@v1
+        with:
+          command: fetch
+      - name: cargo test build
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --tests --release
+      - name: cargo test
+        uses: actions-rs/cargo@v1
         with:
           command: test
-          args: --all-features
+          args: --release
+
+  deny-check:
+    name: cargo-deny check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: download cargo-deny
+      shell: bash
+      env:
+        DVS: 0.3.0-beta
```

Comment:
A bit annoying to update the version of cargo-deny here and then do that in all repos when we change cargo-deny. But I guess if we move this to its own action later we can update it internally in it. So this can be a good start still

Reply:
Yah, the action would probalby just be a docker action, so it's "free" from this config. This was just a stepping stone. ;)